SAMPLES.md

```diff
@@ -0,0 +1,147 @@
+In this page you can find some samples of generated code using swagger-gradle-codegen for every platform:
+
+## `kotlin` platform
+
+### Data Class
+
+The following definition:
```

Comment:
I would suggest to reorganise it a bit in order to have a complete example of swagger specs

You could have the specs at the end of the page as reference for the kotlin code that you're representing

Reply:
Done, now the specs are on top :)